website/docs/en/api/javascript-api.mdx

```diff
@@ -66,6 +70,16 @@ The `run` method is then used to kickstart all compilation work. Upon completion
 The API only supports a single concurrent compilation at a time. When using `run` or `watch`, call `close` and wait for it to finish before calling `run` or `watch` again. Concurrent compilations will corrupt the output files.
 :::
 
+<Collapse>
+  <CollapsePanel
+    className="collapse-code-panel"
```

Comment:
`collapse-code-panel` repeated many times, can we simplify it?

For example, replace it with `type="code"` or something else

Reply:
this is original collapse antd component, class name is just to override its style of padding and margin. we do not need to wrap it as a new component